altapay.php

```diff
@@ -355,6 +355,16 @@ public function renderAddForm()
                 'name' => $filename,
             ];
         }
+
+        $allTerminal = [];
+        $totalTerminals = count(Altapay_Models_Terminal::getTerminals());
+        for ($i = 1; $i <= $totalTerminals; ++$i) {
+            $allTerminal[] = [
+                'id' => $i,
+                'name' => $i,
```

Comment:
Maybe we can use the array key or an object id instead of a generated range of 1-n.